src/DynamoCore/Search/SearchElements/NodeSearchElement.cs

```diff
@@ -21,10 +21,6 @@ public abstract class NodeSearchElement : ISearchEntry, ISource<NodeModel>
         private bool isVisibleInSearch = true;
 
         internal AutoCompletionNodeElementInfo AutoCompletionNodeElementInfo { get; set; } = new AutoCompletionNodeElementInfo();
-        /// <summary>
-        /// Machine Learning related info
-        /// </summary>
-        public AutoCompletionNodeMachineLearningInfo AutoCompletionNodeMachineLearningInfo { get; set; } = new AutoCompletionNodeMachineLearningInfo();
```

Comment:
Moved this property to NodeSearchElementViewModel as setting this info on NodeSearchElement will interfere with the regular search results. This public property was added after 2.16 and should not be a breaking change.